main.nf

```diff
@@ -22,7 +22,15 @@ if (params.validate_params) {
     validateParameters()
 }
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Ensure both --db_profiles and --db_clusters are provided together
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
 
+if ((params.db_profiles && !params.db_clusters) || (!params.db_profiles && params.db_clusters)) {
```

Comment:
You don't have to change this, because it's probably easier to understand the way you have it, but you could do this with an XOR: true only if one of the two is true.

I don't think Python has an actual XOR operator, so it might reduce to something like:

`if bool(params.db_profiles) != bool(params.db_clusters):`

but again, just a comment, not necessary to change.

Reply:
Great suggestion Eric - played around a bit and got it working here: [7d54942](https://github.com/phac-nml/gasnomenclature/pull/29/commits/7d549422c74403bf9ba8b08fcc6923d2396a175f)